tests/hmm/plotting.py

```diff
@@ -739,7 +739,7 @@ def PrintDatacard(categories, event_counts, filenames, ofname):
     elif ("h_peak" in cat.full_name) or ("h_sideband" in cat.full_name):
         dcof.write("R rateParam {0} dy_m105_160_amc 1 \n".format(cat.full_name))           
         dcof.write("R rateParam {0} dy_m105_160_vbf_amc 1 \n".format(cat.full_name))
-        dcof.write("REWZ rateParam {0} ewk_lljj_mll105_mjj160 1 \n".format(cat.full_name))
+        dcof.write("REWZ rateParam {0} ewk_lljj_mll105_160 1 \n".format(cat.full_name))
```

Comment:
I would suggest to remove this line:  

dcof.write("REWZ rateParam {0} ewk_lljj_mll105_160 1 \n".format(cat.full_name))

and add back the 20% norm uncertainty on EWZ because letting it float is a check, not meant for the final fit.